server/src/tenant/schema-generation/graphql-types/connection.graphql-type.ts

```diff
@@ -0,0 +1,24 @@
+import { GraphQLList, GraphQLNonNull, GraphQLObjectType } from 'graphql';
+
+import { PageInfoType } from './page-info.graphql-type';
+
+/**
+ * Generate a GraphQL connection type based on the EdgeType.
+ * @param EdgeType Edge type to be used in the connection.
+ * @returns GraphQL connection type.
+ */
+export const generateConnectionType = <T extends GraphQLObjectType>(
+  EdgeType: T,
```

Comment:
Not a strong opinion here but I think I'd prefer to use ObjectType here as well for the name, this way you won't have to explain why you are removing "Edge" from the name and it's clearer to see that you are simply adding Connection to the ObjectType. Wdyt?

Reply:
I've done that so we can call each of them separately, and avoid nesting call inside each generator